inventory_management_system_api/repositories/manufacturer.py

```diff
@@ -124,7 +124,7 @@ def delete(self, manufacturer_id: str, session: Optional[ClientSession] = None)
         logger.info("Deleting manufacturer with ID: %s from the database", manufacturer_id)
         result = self._manufacturers_collection.delete_one({"_id": manufacturer_id}, session=session)
         if result.deleted_count == 0:
-            raise MissingRecordError(f"No manufacturer found with ID: {str(manufacturer_id)}")
+            raise MissingRecordError(f"No manufacturer found with ID '{str(manufacturer_id)}'")
```

Comment:
Looks like we don't need to cast it to `str` as `f-strings` will take care of that.
```suggestion
            raise MissingRecordError(f"No manufacturer found with ID '{manufacturer_id}'")
```